administrator/components/com_admin/sql/updates/mysql/4.0.0-2019-11-18.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE `#__banners` DROP COLUMN `alias`;
+UPDATE `#__content_types`
+  SET `field_mappings` = REPLACE(`field_mappings`,',"core_alias":"alias"','')
```

Comment:
```suggestion
  SET `field_mappings` = REPLACE(`field_mappings`,'"core_alias":"alias"','"core_alias":"null"')
```
Maybe it is not right to remove the mapping? At other places where a field for a mapping does not exist in the target table, it is done with mapping to "null", see e.g. ` "core_language":"null"` in the field mapping for `com_users.user`.

Reply:
I wonder if we can update this to use `null` value and not `null` string 😕

Reply:
Are there nulls in json?

Reply:
The code checks for "null" as string, as far as I remember from a while ago.

Reply:
And it's done in that way everywhere else where a field has no mapping.

Reply:
JSON does have nulls. that's why i suggested it on another one of your PRs a month or so back. But if every other component is doing it - not something we should start here

Reply:
The suggested code change is not correct. It works correctly as it is

Reply:
If the suggested code is not correct, then it is also not correct at all other places where we have it like this. @wilsonge please check and confirm if so.

Reply:
I am referring to your suggestion to remove the ,
That is completely correct and I doubt that it exists anywhere else

Reply:
Open your eyes: I do not remove the comma.

Reply:
git disagrees - my eyes are open
![image](https://user-images.githubusercontent.com/1296369/69827335-daec9000-120e-11ea-99b4-79e93597024e.png)

Reply:
Then it seems you don't know how git works: When someone suggests a code change, the diff between the change in the PR and the suggestion is shown. Because you were removing the complete mapping, you had to remove the comma, too. Because I don't remove the mapping but change it, I shall not remove the comma. Got it now?

Reply:
P.S. to be more precise: With "I shall not remove the comma" I mean I shall not remove it from the json, and that's why I have to remove it from the search string to be replaced.